Whois/pyproject.toml

```
@@ -8,8 +8,8 @@ package-mode = false
 [tool.poetry.dependencies]
 python = ">=3.10,<3.12"
 pysocks = "^1.6"
-python-whois = "^0.8"
-sekoia-automation-sdk = "^1.13.0"
+python-whois = "^0.9.5"
```

Could you revert the upgrade of `python-whois`? The output format change between the two versions that may impact existing playbook. We have to evaluate this upgrade before.
```suggestion
python-whois = "^0.8"
```